integration/simulation/transaction_injector.go

```diff
@@ -235,7 +235,7 @@ func (m *TransactionInjector) issueInvalidL2Txs() {
 			toWallet = m.rndObsWallet()
 		}
 		var tx types.TxData
-		switch rand.Intn(1) { //nolint:gosec
+		switch rand.Intn(2) { //nolint:gosec
```

Comment:
Is this an unrelated bugfix? Don't want to be fussy but could maybe drop it in a separate PR. Feels like the cleaner thing to do, then if tests start behaving very differently it'll be more obvious whether it was this or the config changes.

Reply:
I would do, but it's breaking the linter, so I have to fix it (the alternative is to suppress it for now, but that feels worse).